html/themes/custom/common_design_subtheme/templates/hr-paragraphs-rss-feed.html.twig

```diff
@@ -16,11 +16,11 @@
         {% if item.description|length > 0 %}
           {% if options.display_each_item_full %}
             <div class="external-feed--item-description--full">
-              {{ item.description }}
+              {{ item.description|striptags|raw }}
             </div>
           {% else %}
             <div class="external-feed--item-description--teaser">
-              {{ item.description|slice(0, 200)|raw }}
+              {{ item.description|slice(0, 200)|striptags|raw }}
```

Comment:
`Striptags` should probably appear before the slice, otherwise you may have truncated and invalid HTML content.